clang/test/Format/dump-config.cpp

```diff
@@ -0,0 +1,3 @@
+// RUN: clang-format -dump-config 2>&1 | FileCheck %s
```

Comment:
Can you also add a test with `clang-format --assume-filename=foo.m -dump-config 2>&1 | FileCheck %s`?

It should result in `Language: ObjC`.

Reply:
Added to dump-config-objc-stdin.m instead.